steam.py

```diff
@@ -125,17 +125,11 @@ def sync_wishlist_file():
 
 def delete_game(app_tuple):
     print("Entering the delete_games function")
-    # Check if game ID exists
-    # for app in games_list_json
-    for app in wishlist_json: 
-        if app_tuple[0] == app["appid"]:
-            print(f"\"{app_tuple}\" found, will now be deleted")
-            #Deleting if game found in games_list_json
-            wishlist_json.remove(app)
-            sync_wishlist_file()
-            print(f'Games being tracked after deletion are: {wishlist_json}')
-            return
-    return
+    app_to_delete = f'{app_tuple[0]}'
+    del wishlist_json[app_to_delete]
+    sync_wishlist_file()
+    reply = 'Game was successfully deleted, it is no longer being tracked'

```

Comment:
Steam file shouldn't be responsible for knowing what a "reply" is. Instead it should tell the caller the result of the deletion, and let the caller decide how to proceed. 

In this case the steam file really only knows about managing local wishlist and performing fetches on the Steam API. Client.py should always be responsible for forming replies. 
This type of structure would allow us to call the delete function from other places that might not know what to do with a String reply.

Reply:
Gotcha, will make a class to pass back to steam.py to build the reply in there

Reply:
You could probably use a boolean (true, false) here since it either did delete, or didn't exist. I don't think we really have a case where the actual delete fails